lib/block_view_test.go

```diff
@@ -6642,9 +7246,9 @@ func TestBitcoinExchangeGlobalParams(t *testing.T) {
 
 			// When we hit the rate update, populate the placeholder.
 			if ii == rateUpdateIndex {
-				newUSDCentsPerBitcoin := uint64(27000 * 100)
+				newUSDCentsPerBitcoin := int64(27000 * 100)
```

Comment:
I had to change the updateGlobalParams helper function to accept an int instead of a uint so that I could specify values of "-1" meaning, "do not update."